sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,4 @@
+-- Ironhand Guardion ScriptName
+SET @ENTRY := 8982;
+UPDATE `creature_template` SET `AIName`='', ScriptName='npc_ironhand_guardian' WHERE `entry`=@ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
```

Comment:
No need of "AND `source_type`=0" if you are moving the mob to c++.

Reply:
I'm unsure why that is? I though that I in principle could hit objects (such as gameobjects or areatriggers) in the smart_script table. So the entry number is unique across the different types?

Reply:
`smart_scripts`.`entryorguid` is supposed to be unique to avoid conflicts.
(`guid` is obvious, although TC prefers to keep GOs, creatures and area triggers separated by entry/ID too.)

Reply:
... just F.Y.I. regarding possible SAI script rows, if there are `entryorguid`s with 00, 01, 02 etc. in the table, check for it with % and text ( `' '` ) :
```sql
SELECT * FROM `smart_scripts` WHERE `entryorguid` LIKE '8982%';
```
(currently, there are only 2 lines for `entryorguid` = 8982 in TDB 335.64)